test/test_torch.py

```diff
@@ -17373,6 +17375,67 @@ def test_hypot(self, device, dtype):
                 expected = np.hypot(input[0].cpu().numpy(), input[1].cpu().numpy())
             self.assertEqual(actual, expected)
 
+    def _helper_test_igamma(self, loglo, loghi, device, dtype):
+        exp1 = 2.71828182846
+        vec1 = torch.logspace(loglo, loghi, steps=500, base=exp1,
+                              dtype=torch.float64, device=device).unsqueeze(-1)
+        vec1 = vec1.to(dtype)
+        inputs = [
+            (vec1, vec1.transpose(0, 1)),
+            (vec1, vec1),  # for large number, it should approach 0.5
+            (vec1, 0.5 * vec1),  # test for considerable ratio
```

Comment:
Make the first input discontiguous for this test, too.

Reply:
What do you mean by "discontiguous"?

Reply:
Such that not all the elements of the tensor are sequential in memory. This could be done, for example, by taking every other element of vec1:

```
vec1[::2]
```

Reply:
Ah, sure!